deps/build.jl

```diff
@@ -2,14 +2,58 @@
 import JSON
 using Compat
 
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
+if isdir(abspath(dirname(@__FILE__), "..", "..", "Conda"))
+    Conda_installed = true
+    import Conda
+    Conda.add("ipython-notebook=3.1")
+else
+    Conda_installed = false
+end
+
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
 # Make sure Python uses UTF-8 output for Unicode paths
 ENV["PYTHONIOENCODING"] = "UTF-8"
 
-include("jupyter.jl")
-const jupyter, jupyter_vers = find_jupyter()
+if Conda_installed
+    condapath=dirname(Conda.conda)
+    const jupyter, jupyter_vers = try
+        joinpath(condapath,"jupyter"),convert(VersionNumber, chomp(readall(`$(joinpath(condapath,"jupyter")) kernelspec --version`)))
+    catch e1
+        try
+            joinpath(condapath,"ipython"),convert(VersionNumber, chomp(readall(`$(joinpath(condapath,"ipython")) --version`)))
```

Comment:
What is the point of this?  Won't Jupyter always be installed, assuming you do `Conda.add("jupyter")`?